civihr_default_theme/scss/partials/_button.scss

```diff
@@ -0,0 +1,15 @@
+.btn-default {
+  border-color: $crm_gray-border;
+  color: $crm_gray-border;
+
+  &.active {
+    border-color: $crm_gray-border;
+    background: $crm_gray-border !important;
```

Comment:
It was added to override the hover state color when item is active. Modified to
` &.active,
  &.active:hover {`